### PR TITLE
add support for new config file in debian installs

### PR DESCRIPTION
**Pull request by bbaldino**
The intent here is to generate an empty `jvb.conf` if one doesn't exist, and pass this config file (via `-Dconfig.file`) to the bridge when starting up so that debian installs can support the new configuration.

I didn't go so far as to migrate any configurations put in `sip-communicator.properties` into the new config, as the logic in this script is already pretty complex (already supports migrating from an old, _old_ config file). Maybe there's some code that can be cleaned up/removed there for the older cases?

### CODE REVIEW

File: debian/postinst [hunk 1]
```diff
@@ -117,6 +120,12 @@ case "$1" in
             echo "org.jitsi.videobridge.xmpp.user.shard.MUC_NICKNAME=$(uuidgen)" >> $NEW_JITSI_CONFIG
         fi
 
+        if [ ! -f $HOCON_CONFIG ]; then
+            echo "Generating an empty hocon config"
+            echo "videobridge {
+            }" >> $HOCON_CONFIG
```
Comment (JonathanLennox) [inline debian/postinst:126]: If the file doesn't exist you don't need `>>`, just `>`, though I suppose it doesn't matter.

Should the file have a comment at the top giving a ref to the appropriate point in the JVB docs for its contents and syntax?

Comment (bbaldino) [reply debian/postinst:126]: I think it'll be the other way around: docs will point to this file.

**Comment by damencho** [comment]
@bbaldino The parameter `-Dconfig.file...` will be added only for new install and the empty file will be created for old and new installs, is this the intended behavior?

**Comment by bbaldino** [comment]
> @bbaldino The parameter `-Dconfig.file...` will be added only for new install and the empty file will be created for old and new installs, is this the intended behavior?

Nope, that wasn't intended.  I think what we want to do both of those things for both old and new.

**Comment by damencho** [comment]
I see. So you can be checking the config file for existing of `-Dconfig.file=` and then replace `JAVA_SYS_PROPS="` with `JAVA_SYS_PROPS ="-Dconfig.file= ` (with extra space at the end).
And that will cover old and new, I think.

**Comment by damencho** [comment]
I hope sed with `|` as a delimiter is available on all modern distros

**Review by damencho — APPROVED** [review]

**Pull request merged**